ddtrace/llmobs/_integrations/anthropic.py

```diff
@@ -143,24 +143,28 @@ def _extract_input_message(self, messages, system_prompt=None):
     def _extract_output_message(self, response):
         """Extract output messages from the stored response."""
         output_messages = []
-        content = _get_attr(response, "content", None)
+        content = _get_attr(response, "content", "")
         role = _get_attr(response, "role", "")
 
         if isinstance(content, str):
-            return [{"content": content, "role": role}]
+            return [{"content": content, "role": role, "tool_calls": []}]
 
         elif isinstance(content, list):
             for completion in content:
+                tool_calls_info = []
                 text = _get_attr(completion, "text", None)
                 if isinstance(text, str):
-                    output_messages.append({"content": text, "role": role})
+                    output_messages.append({"content": text, "role": role, "tool_calls": tool_calls_info})
```

Comment:
Do we need to specify `"tool_calls"` as a field if there is no tool calls to report?

Reply:
for the anthropic case, we can get rid of this. Do we want to update openai to also not return "tool_calls" as a field if there is no tool calls to report? Right now the logic for openai is returning an empty list of "tool_calls" if only content is present.